lib/Params.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+// Copyright 2022. Plesk International GmbH. All rights reserved.
+
+namespace WHMCS\Module\Server\Plesk360Monitoring;
+
+class Params
+{
+    public const PLAN_ID = 'configoption1';
+    public const SERVERS = 'configoption2';
+    public const WEBSITES = 'configoption3';
+
+    public const SERVER_SCHEME = 'serverhttpprefix';
+    public const SERVER_HOST = 'serverhostname';
+    public const SERVER_PORT = 'serverport';
+    public const SERVER_USERNAME = 'serverusername';
+    public const SERVER_PASSWORD = 'serverpassword';
+
+    private const OPTION_SERVERS = 'Servers';
+    private const OPTION_WEBSITES = 'Websites';
+
+    public static function serverAllowance(array $params): int
+    {
+        $servers = (int)$params[self::SERVERS];
```

Comment:
undefined index?

Reply:
No, we define these in `p360monitoring_ConfigOptions`